src/explore-education-statistics-admin/src/pages/release/data/components/ApiDataSetLocationMapModal.module.scss

```diff
@@ -0,0 +1,14 @@
+@import '~govuk-frontend/dist/govuk/base';
```

Comment:
Was just wondering something about the govuk design system - do you have to load this base file into every CSS file where you want to use govuk variables? Just pondering on how that affects performance & whatnot

Reply:
We have to import in any scss module where we need access to the govuk variables etc, but I don't think it effects performance.